woocommerce/woocommerce-android
# Add is editable field

‼️ Don't merge until [FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2435) changes are merged 

Closes: #6649
Closes #6650

### Description
Add the `is_editable` field to the Order model. Now we can check if an order is editable based on this new field. 

### Testing instructions

TC1
1. Select a site using WooCommerce < 6.6
2. Open the Orders tab in the app
3. Open Flipper or the App inspection
4. Check `isEditable` column exists

TC2
1. Select a site using WooCommerce > 6.6
2. Open the Orders tab in the app
3. Open Flipper or the App inspection
4. Check `isEditable` column exists

TC3
1. Install the version from `trunk`
2. Open the Orders tab in the app
3. Install the version from this branch
4. Check the app don't crash because of any DB migration issues

### Images/gif
<img width="1363" alt="Screen Shot 2022-06-01 at 15 35 01" src="https://user-images.githubusercontent.com/18119390/171478394-621fa5be-dd2a-4253-908a-fe1265e74f21.png">
